pkg/kube/unstructured_test.go

```diff
@@ -37,9 +38,9 @@ func (s *UnstructuredSuite) TestFetch(c *C) {
 	gvr := schema.GroupVersionResource{
 		Group:    "",
 		Version:  "v1",
-		Resource: "serviceaccounts",
+		Resource: "service",
```

Comment:
```suggestion
		Resource: "services",
```
I think it's services instead of service 🤔